shared-ts/siriTypes.zod.ts

```diff
@@ -83,6 +83,7 @@ export const networksSchema = z.object({
                 z.object({
                     AffectedOperator: affectedOperatorSchema,
                     LineRef: z.string(),
+                    PublishedLineName: z.string().optional(),
```

Comment:
I don't think this should be optional as it is a required field now as part of the SIRI updates

```suggestion
                    PublishedLineName: z.string(),
```

Reply:
as discussed